roles/confluent.variables/vars/main.yml

```diff
@@ -251,14 +264,14 @@ schema_registry_properties:
       inter.instance.protocol: "{{schema_registry_http_protocol}}"
       kafkastore.bootstrap.servers: "{{ groups['kafka_broker'] | default(['localhost']) | join(':' + kafka_broker_listeners[schema_registry_kafka_listener_name]['port']|string + ',') }}:{{kafka_broker_listeners[schema_registry_kafka_listener_name]['port']}}"
   ssl:
-    enabled: "{{schema_registry_ssl_enabled}}"
+    enabled: "{{schema_registry_ssl_enabled or mds_broker_listener.ssl_enabled|bool}}"
```

Comment:
I think this or should be using mds_ssl_enabled not the listener because SR does not talk to the listener.
Also there should be a way to have SR talk to mds when mds has ssl on without turning on ssl or SRs rest endpoint (id hope)

Reply:
Great point about mds_ssl_enabled.  So this doesn't enforce Schema Registry to use SSL on it's endpoint, it listens for plaintext only.  We would have to specify and HTTPS listener for it to do so.  When I tested this, I still had to include the TLS properties for the listener in order for schema registry to work with TLS on the MDS on the remote cluster.  I don't understand why that's the case, but its the only way I could get it working.